scipy_doctest/impl.py

```diff
@@ -518,7 +519,10 @@ def get_history(self):
         an object, and the value is a tuple of the numbers of examples which
         failed and which were tried.
         """
-        return self._name2ft
+        if version_info >= (3, 13):
+            return self._stats
```

Comment:
```suggestion
            # cf https://github.com/python/cpython/commit/4f9b706c6f5d4422a398146bfd011daedaef1851
            return self._stats[:2]
```

Reply:
https://github.com/scipy/scipy_doctest/issues/191#issuecomment-2776223103

Reply:
Discussion in https://github.com/scipy/scipy_doctest/issues/191#issuecomment-2776730298.